index.bs

```diff
@@ -115,8 +115,100 @@ The <dfn for="a" element-attr>attributionreportto</dfn> attribute optionally dec
 The <dfn for="a" element-attr>attributionexpiry</dfn> attribute optionally defines the amount
 of time in milliseconds the attribution source should be considered for reporting.
 
-Issue: Need monkey patches passing attribution source in navigation, and a mechanism
-for validating the resulting document matches the attributiondestination.
+<h3 id="monkeypatch-navigation">Navigation</h3>
+
+The user agent must verify that an [=attribution source=] associated with a navigation
```

Comment:
Don't use "must" in introductory paragraphs like this, since that introduces a normative requirement. You could say

```suggestion
This section ensures that an [=attribution source=] associated with a navigation
```